apps/api/src/domain/models/audio-item/errors/ConflictingLineItems.error.ts

```diff
@@ -0,0 +1,12 @@
+import { InternalError } from '../../../../lib/errors/InternalError';
+import { TranscriptItem } from '../entities/transcript-item.entity';
+
+export class ConflictingLineItemsError extends InternalError {
+    constructor(newLineItem: TranscriptItem, conflictingExistingItems: TranscriptItem[]) {
+        super(
+            `You cannot add the new line item: ${newLineItem.toString()} to this transcript as its time stamp conflicts with those of the following existing line item(s):\n${conflictingExistingItems
```

Comment:
Determine whether the (s) is required based on `conflictingExistingItems.length`.

Reply:
done